math/eratosthenes.cpp

```diff
@@ -1,118 +0,0 @@
-/**
- * @file
- * @brief [The Sieve of
- * Eratosthenes](https://en.wikipedia.org/wiki/Sieve_of_Eratosthenes)
```

Comment:
why was this deleted

Reply:
it doesn't look deleted to me, just has the addition of a short description of what the algorithm does right? Well, the file shows it as deleted but then re-added with the addition.

Reply:
both of these files have different file names and are not the same algoritms